validation-test/Sema/type_checker_perf/fast/rdar19181998.swift.gyb

```diff
@@ -1,4 +1,4 @@
-// RUN: %scale-test --invert-result --begin 1 --end 4 --step 1 --select incrementScopeCounter %s
+// RUN: %scale-test --begin 1 --end 30 --step 1 --select incrementScopeCounter %s
```

Comment:
Is this because of the change to move the check for bound generic types into the binding generation?

Reply:
Yes, that's right. Because these bindings are not going to be recorded instead of skipped it changes ordering a bit.